src/figuresClasses.ts

```diff
@@ -2,10 +1,99 @@
+type Shape = string | 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Rectangle implements Figure {
+  shape: string = 'rectangle';
+
+  color: 'red' | 'green' | 'blue';
```

Comment:
[CODE STYLE] - The 'color' property type is repeated across multiple classes. It would be better to use the 'Color' type that was declared at the top to maintain consistency and reduce redundancy.